### PR TITLE
[ES] Quote

### PR DESCRIPTION
**Pull request by luischos**
es-35
es-39
es-37

### CODE REVIEW

**Comment by qwrtln** [comment]
Please replace tabs with spaces, as the linter suggests.

**Review by qwrtln — APPROVED** [review]

**Review by qwrtln — CHANGES_REQUESTED** [review]
Fix linters

**Review by qwrtln — APPROVED** [review]

**Review by qwrtln — CHANGES_REQUESTED** [review]
checking

**Review by qwrtln — APPROVED** [review]

**Review by qwrtln — CHANGES_REQUESTED** [review]
Sorry, testing new mechanism for PR reviews. Please address the issues. This one if for real 😆

File: main_es.tex [hunk 1]
```diff
@@ -22,6 +25,21 @@
   Todo el mundo es bienvenido a contribuir, hacer cambios y corregir errores.
   Si simplemente quieres dejar comentarios, por favor hazlo en el \href{https://boardgamegeek.com/thread/3235221/rule-book-rewrite-project}{hilo original de BoardGameGeek}.
 }
+
+\newcommand{\heegusquote}{
+  \textit{
+    Aunque se ha puesto mucho empeño en complacer a los fans, sobre todo en lo que se refiere al aspecto visual, la jugabilidad no hace más que estorbar.
```
Comment (qwrtln) [inline main_es.tex:31]:
```suggestion
  \textit{Aunque se ha puesto mucho empeño en complacer a los fans, sobre todo en lo que se refiere al aspecto visual, la jugabilidad no hace más que estorbar.
```
I suggest a slight change like so.

File: main_es.tex [hunk 2]
```diff
@@ -5,6 +5,9 @@
 \usepackage[T1]{fontenc}
 \usepackage[spanish]{babel}
 
+\def\sections{sections/translated/es}
+\def\lang_header_adjustment{0px}
+
```
[the remark this answers is not in the document]
Comment (luischos) [reply main_es.tex:10]: What is the problem? I see this:

**Comment by luischos** [comment]
Anything else to fix @qwrtln ?

File: main_es.tex (hunk 2 above)
Comment (qwrtln) [inline main_es.tex:10]:
```suggestion
```
Then it's not necessary.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**